packages/driver/src/cy/commands/actions/select.ts

```diff
@@ -104,13 +104,16 @@ export default (Commands, Cypress, cy) => {
         $errUtils.throwErrByPath('select.invalid_multiple')
       }
 
+      const subjectChain = cy.subjectChain()
+
       const getOptions = () => {
+        options.$el = cy.getSubjectFromChain(subjectChain)
```

Comment:
If I'm understanding correctly, we are requerying each time a chained command executed, not when it's retried?